src/explore-education-statistics-frontend/src/middleware/pages/redirectPages.ts

```diff
@@ -26,9 +26,6 @@ const redirectPatterns: URLPattern[] = [
   new URLPattern({
     pathname: `/find-statistics/:${publicationSlugKey}/:${releaseSlugKey}/(data-guidance/?|prerelease-access-list/?|.{0})?`,
   }),
-  new URLPattern({
-    pathname: `/data-tables/:${publicationSlugKey}{/}?`,
-  }),
```

Comment:
I'm wondering if removing this is correct because this looks like it's covering the route `data-tables/{publicationSlug}` AFTER you've selected a publication in step 1 of the table tool, which I guess could be shared/bookmarked.

Or is this still covered by the pattern below?

Reply:
You're right, I've added this back in :)